src/cli/cli.ts

```diff
@@ -9,7 +10,7 @@ export async function run(): Promise<void> {
     describe: "add a plugin",
   }).argv;
 
-  const mainArgv = commandDir(yargs.help(), "commandModules").demandCommand();
+  const mainArgv = commandDir(yargs.help(), Path.join(__dirname, "commandModules")).demandCommand();
```

Comment:
This seems correct. Has this been not working or is this just an improvement?

Reply:
It was using a relative path before, which breaks now that the commandDir file has been moved.